tensorflow/lite/kernels/unpack.cc

```diff
@@ -79,6 +80,11 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
     TfLiteIntArray* copied_output_shape = TfLiteIntArrayCopy(output_shape);
     TfLiteTensor* output = GetOutput(context, node, i);
     TF_LITE_ENSURE_EQ(context, output->type, input->type);
+    // Guarantee input/output quantization params match as we do not support
+    // unpacking quantized tensors.
```

Comment:
This comment is slightly confusing, checking quantization match makes sense, but "we do not support unacking quantized tensors" doesn't sound right, as that's exactly what we're doing? Did you mean rescaling quantized tensors?